packages/runtime/LICENSE

```diff
@@ -1 +1,28 @@
-../../LICENSE
\ No newline at end of file
+The ALGOB is licensed under the Apache-2.0 license as defined in the
```

Comment:
Please copy the license file from the root directory.

Reply:
so we should just use the Apache-2.0 license.

Reply:
ok